warehouse/query-core/src/main/java/datawave/query/jexl/lookups/FieldExpansionIndexLookup.java

```diff
@@ -0,0 +1,172 @@
+package datawave.query.jexl.lookups;
+
+import java.util.HashSet;
+import java.util.Map.Entry;
+import java.util.Set;
+import java.util.concurrent.Callable;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Future;
+import java.util.concurrent.atomic.AtomicLong;
+
+import org.apache.accumulo.core.client.IteratorSetting;
+import org.apache.accumulo.core.client.Scanner;
+import org.apache.accumulo.core.data.Key;
+import org.apache.accumulo.core.data.PartialKey;
+import org.apache.accumulo.core.data.Range;
+import org.apache.accumulo.core.data.Value;
+import org.apache.hadoop.io.Text;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.base.Joiner;
+import com.google.common.base.Preconditions;
+
+import datawave.core.iterators.FieldExpansionIterator;
+import datawave.query.config.ShardQueryConfiguration;
+import datawave.query.tables.ScannerFactory;
+import datawave.util.time.DateHelper;
+
+/**
+ * An {@link IndexLookup} that wraps {@link FieldExpansionIterator}.
+ */
+public class FieldExpansionIndexLookup extends AsyncIndexLookup {
+
+    private static final Logger log = LoggerFactory.getLogger(FieldExpansionIndexLookup.class);
+
+    protected String term;
+    protected Future<Boolean> timedScanFuture;
+    protected AtomicLong lookupStartTimeMillis = new AtomicLong(Long.MAX_VALUE);
+    protected CountDownLatch lookupStartedLatch;
+    protected CountDownLatch lookupStoppedLatch;
+
+    private Scanner scanner;
+
+    public FieldExpansionIndexLookup(ShardQueryConfiguration config, ScannerFactory scannerFactory, String term, Set<String> fields,
+                    ExecutorService execService) {
+        super(config, scannerFactory, true, execService);
+        this.term = term.toLowerCase();
```

Comment:
why lower case here?  normalizers do not necessarily lower case values.

Reply:
Right you are...I need to revisit this case. The previous implementation was doing a just-in-time expansion for normalization which implies the ExpandMultiNormalizedTerms doesn't handle unfielded terms well..

Reply:
I went on a journey through the query planner...